goreman_test.go

```diff
@@ -1,18 +1,55 @@
-// Windows doesn't have sleep.exe which we use for making the subprocesses stay
-// alive for a certain amount of time.
-
-// +build !windows
-
 package main
 
 import (
 	"context"
 	"io/ioutil"
 	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
 	"testing"
 	"time"
 )
 
+var sleep string
+
+func TestMain(m *testing.M) {
+	var dir string
+	var err error
+	sleep, err = exec.LookPath("sleep")
+	if err != nil {
+		if runtime.GOOS == "windows" {
```

Comment:
You can combine these into one line if you want

Reply:
* err == nil and linux => nothing to do
* err != nil and linux => panic
* err == nil and windows => nothing to do
* err != nil and windows => fallback

So nested-if is clean